site/daemon.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Weave Flux Daemon
+menu_order: 90
+---
+
+# Weave Cloud Deploy
+
+Flux daemon (fluxd, aka flux agent) is at the heart of Weave Cloud Deploy, functionality allowing automation of application deployments and version control of cluster configuration.
+Version controlling of cluster manifests provides reproducibility and a historical trail of events.
+
+## Flux daemon responsibilities
+
+    A) Continuous Deployment
+	    1.
+	    Flux daemon monitors container registry for running container image updates.
+        Detection of an image change (running container image tag vs container
+        registry image tag) triggers k8s manifest update, which is committed to the
+        user git repository, then deployed to the Kubernetes cluster.
+
+	    2.
+	    Flux daemon monitors user git repo Kubernetes manifests for changes, which it
+        then deploys to the cluster.
+
+    B) Deployment approaches
+        1.
+        Automate vs Deautomate
+            Deployment happens automatically when a new image tag is detected.
+            Deautomated deployment will not proceed until manually released (through
+            the UI or the CLI tool fluxctl).
+            
+        2.
+        Lock vs Unlock
+            Deployment is pinned to a particular image tag. New deployment will not
+            proceed upon triggered release.
+    
+# Flags
+
+fluxd requires setup and offers customization though a multitude of flags.
+
+|flag                    | default                       | purpose |
+|------------------------|-------------------------------|---------|
+|--listen -l             | :3030                         | Listen address where /metrics and API will be served|
+|--kubernetes-kubectl    |                               | Optional, explicit path to kubectl tool|
+|--version               | false                         | Get version number|
+|**Git repo & key etc.** |                              ||
+|--git-url               |                               | URL of git repo with Kubernetes manifests; e.g., git@github.com:weaveworks/flux-example|
```

Comment:
It makes the markdown noisier, but it might end up easier to read if code samples and example inputs or values are in monospace, i.e., `backtick-quoted`.

Reply:
Done